css/contact.css

```diff
@@ -73,3 +73,51 @@ label {
   width: 200px;
   height: 60px;
 }
+
+@media(max-width: 700px){
+  *{
+  padding: 10px;
```

Comment:
this (*) will put padding on every element. nothing wrong with doing it. just be aware